releasenotes/notes/update-cr-hamtomo-with-duration-380da3452045cd0c.yaml

```diff
@@ -0,0 +1,30 @@
+---
+features:
+  - |
+    :class:`.CrossResonanceHamiltonian` experiment and its subclass now accept
+    ``durations`` with default values. Note that values should be provided in
+    units of sec rather than samples, and must include pulse ramps at edges.
+    Default values with linear increment are generated according to new experiment options,
+    ``durations_min``, ``durations_max`, and ``durations_num``, when the durations
```

Comment:
```suggestion
    ``min_duration``, ``max_duration`, and ``num_durations``, when the durations
```

Reply:
done in https://github.com/Qiskit/qiskit-experiments/pull/933/commits/1f1bd5a838aed7a50d1f6a42fe5b71d592c52396